src/Features/Lsif/Generator/Generator.cs

```diff
@@ -150,24 +154,25 @@ public async Task GenerateForProjectAsync(Project project, GeneratorOptions opti
             GeneratorOptions options,
             IResultSetTracker topLevelSymbolsResultSetTracker,
             ILsifJsonWriter lsifJsonWriter,
-            IdFactory idFactory)
+            IdFactory idFactory,
+            CancellationToken cancellationToken)
         {
             // Create and keep the semantic model alive for this document.  That way all work/services we kick off that
             // use this document can benefit from that single shared model.
-            var semanticModel = await document.GetRequiredSemanticModelAsync(CancellationToken.None);
+            var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken);
 
-            var (uri, contentBase64Encoded) = await GetUriAndContentAsync(document).ConfigureAwait(false);
+            var (uri, contentBase64Encoded) = await GetUriAndContentAsync(document);
```

Comment:
💭 This one looks like it's missing a cancellation token. Might be better to add it as an unused parameter than to leave it for future signature change.

Reply:
yup!  added.